Automattic/woocommerce-payments
# Added learn more URL to deposits page once

Fixes #5370 

#### Changes proposed in this Pull Request
This PR adds "learn more" URL next to "Change" url on deposits screen.









#### Testing instructions

Make sure that you go to the linked issue and pay attention what needs to be changed and make sure that all changes are visible. Make sure that you test this with older account since you won't see the change if your account is new one since deposits are locked for new accounts.

-------------------


**Post merge**
